metapool/literals.py

```diff
@@ -0,0 +1,112 @@
+# I am aware that "having a global constants class" is largely regarded as
+# an antipattern in Python because it increases dependence between the
+# modules using it and keeps pieces tightly coupled that should be loosely
+# coupled.  However, the current codebase has some literal strings repeated
```

Comment:
For my edification, how worried are we about recompile time?

Reply:
I'm not too worried that it will drastically impact recompile times; however if it did that would impact developing and running tests, which wouldn't be great. I haven't seen this antipattern used in a while so I'm curious to see if it has any real-world effect!